<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Processor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>244</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>File Path / RTSP Url / Camera Index :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="videoEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>610</y>
      <width>142</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>610</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>661</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Video">
     <attribute name="title">
      <string>Video</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>401</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="upButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>390</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>UP</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="leftButton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>420</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Left</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rightButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>420</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Right</string>
      </property>
     </widget>
     <widget class="QPushButton" name="downButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>450</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cclockWiseButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>520</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>C.Clockwise</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clockWiseButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>480</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Clockwise</string>
      </property>
     </widget>
     <widget class="QPushButton" name="extendButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>480</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Extend</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="reduceButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>520</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Reduce</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="recBtn">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>510</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>REC</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LineBtn">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>510</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Linesearch</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="RGB">
     <attribute name="title">
      <string>RGB</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>621</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>510</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="RGBCheckLayout">
       <item>
        <widget class="QCheckBox" name="RedCheck">
         <property name="text">
          <string>Red</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="GreenCheck">
         <property name="text">
          <string>Green</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="BlueCheck">
         <property name="text">
          <string>Blue</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Detecting">
     <attribute name="title">
      <string>Detecting</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>621</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>510</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Detecting</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="HistogramLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Histogram</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="redHisto"/>
     </item>
     <item>
      <widget class="QGraphicsView" name="greenHisto"/>
     </item>
     <item>
      <widget class="QGraphicsView" name="blueHisto"/>
     </item>
    </layout>
   </widget>
   <zorder>label</zorder>
   <zorder>videoEdit</zorder>
   <zorder>startBtn</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>tabWidget</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
